tests/kubeless-test-client/k8s.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: gateway.kyma.cx/v1alpha2
+kind: Api
```

Comment:
Do you really need to expose your service? Is it called from outside of cluster?

Reply:
Yes. In the test we want to ensure that the kubeless controller creates the function correctly and that it is available via api/ingress. This is short-lived and removed at the end of the test.